### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by Skyllarr**

### CODE REVIEW

File: undertow-standalone-jaspi/pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <version.io.undertow>2.0.13.Final</version.io.undertow>
+        <version.io.undertow>2.2.15.Final</version.io.undertow>
```
Comment (fjuma) [inline undertow-standalone-jaspi/pom.xml:29]: Just noticed WildFly Core is on 2.2.18.Final. Might be good to update to that version here too so we don't need to do another round of updates again.

Comment (Skyllarr) [reply undertow-standalone-jaspi/pom.xml]: Thank you for noticing. Fixed

Comment (Skyllarr) [reply undertow-standalone-jaspi/pom.xml]: Actually I made a mistake, just a sec

Comment (Skyllarr) [reply undertow-standalone-jaspi/pom.xml]: @fjuma Fixed

Comment (fjuma) [reply undertow-standalone-jaspi/pom.xml]: Thanks!

**Pull request merged**